### ISSUE TITLE
iPAS AI 應用規劃師科目3練習 - 2026-08-14

### ISSUE THREAD

**Issue by JohnnyLai92** [JohnnyLai92/ipas#9]
我已讀取 `docs` 的出題指南與歷次錯題分析（作為出題來源與難度校準）。以下是可直接貼入 GitHub Issue 的 Markdown 內容（我未修改任何檔案）。

---

# iPAS AI 應用規劃師（中級）科目3 練習 — 2026-08-15

## 練習說明

本次練習共 **10 題**，涵蓋科目3「機器學習技術與應用」四大範圍（L231～L234），題型為 **單選、多選、情境題與程式判讀混合**。

- 請直接在下方「作答格式」區塊 **回覆 1～10 題的答案**（單選填一個字母、多選填多個字母）。
- 每題請附上 **一句簡短理由**，方便後續批改與錯題分析。
- 本次題目 **不揭露正確答案**；作答完成後依「批改說明」操作即可。

---

### 第 1 題〔單選．情境題〕
**考點：監督式 vs 非監督式學習判斷（L23201）**

某電商平台握有大量「未標記」的顧客交易紀錄，想自動把顧客分成數個消費行為相近的群體，事前並不知道應該分成幾群、也沒有任何群體標籤。下列哪一種方法最適合？

- (A) Logistic Regression 分類
- (B) K-Means 分群
- (C) Linear Regression 回歸
- (D) 以人工預先標好群別後再訓練 Decision Tree

---

### 第 2 題〔單選〕
**考點：分類 vs 回歸任務辨識（L23201）**

下列四個 AI 應用情境中，哪一個屬於 **回歸（Regression）** 問題？

- (A) 判斷一封 email 是否為垃圾郵件
- (B) 預測某間房屋下個月的成交價格
- (C) 辨識影像中的動物是貓還是狗
- (D) 判斷信用卡交易是否為盜刷

---

### 第 3 題〔單選．情境題〕
**考點：過擬合診斷（L23302）**

某分類模型的 Training Accuracy 為 98%，但 Validation Accuracy 只有 71%，兩者落差很大。這最可能代表什麼狀況？

- (A) Underfitting（欠擬合）
- (B) Overfitting（過擬合）
- (C) 資料量太少導致無法訓練
- (D) 模型已達到最佳泛化能力

---

### 第 4 題〔單選．計算題〕
**考點：混淆矩陣與 Precision 計算（L23303）**

某二元分類模型的混淆矩陣為：**TP = 60、FP = 15、FN = 25、TN = 100**。請問此模型的 **Precision（精確率）** 約為多少？

- (A) 0.60
- (B) 0.71
- (C) 0.80
- (D) 0.86

---

### 第 5 題〔單選．情境題〕
**考點：評估指標選擇（Recall vs Precision）（L23303）**

某醫院要建立「癌症早期篩檢」模型，最不希望發生的情況是「真的罹病卻被判為健康而漏掉」。在此情境下，應優先重視並提升哪一項指標？

- (A) Precision
- (B) Recall
- (C) 整體 Accuracy
- (D) 訓練速度

---

### 第 6 題〔單選．情境題〕
**考點：深度學習應用策略 / Transfer Learning（L23203）**

某工廠只有 2,000 張已標記的瑕疵產品圖片，GPU 資源有限，卻希望快速建立一個堪用的影像分類模型。下列策略何者最適合？

- (A) 從零開始訓練一個超大型 CNN
- (B) 採用 ImageNet 等預訓練模型進行 Transfer Learning + Fine-tuning
- (C) 改用 K-Means 分群取代影像分類
- (D) 改用 Linear Regression 進行影像分類

---

### 第 7 題〔多選〕
**考點：過擬合改善方法（L23304）**

下列哪些做法「有助於改善過擬合（Overfitting）」？（**多選，請選出所有正確項**）

- (A) 增加訓練資料或使用 Data Augmentation
- (B) 加入 Regularization（L1／L2）或 Dropout
- (C) 大幅增加網路層數與參數量
- (D) 使用 Early Stopping
- (E) 適度降低模型複雜度

---

### 第 8 題〔單選．程式判讀．情境題〕
**考點：資料前處理與 Data Leakage（L23301）**

某工程師寫下如下流程處理資料：

```python
from sklearn.preprocessing import StandardScaler
from sklearn.model_selection import cross_val_score

scaler = StandardScaler()
X_scaled = scaler.fit_transform(X)          # 對「全部資料」計算平均值與標準差
scores = cross_val_score(model, X_scaled, y, cv=5)
```

此流程最可能造成什麼問題？

- (A) 沒有問題，這是標準且推薦的做法
- (B) 發生 Data Leakage，因為 scaler 用到了驗證折的統計資訊，評估結果會過度樂觀
- (C) 一定會造成 Underfitting
- (D) `cross_val_score` 無法在縮放後的資料上執行

---

### 第 9 題〔單選．程式判讀〕
**考點：神經網路 Linear Layer 與 Tensor Shape（L23203／L23102）**

執行下列 PyTorch 程式：

```python
import torch
import torch.nn as nn

model = nn.Linear(4, 2)
x = torch.randn(10, 4)
y = model(x)
print(y.shape)
```

輸出的 `y.shape` 為何？

- (A) `torch.Size([10, 4])`
- (B) `torch.Size([4, 2])`
- (C) `torch.Size([10, 2])`
- (D) `torch.Size([2, 10])`

---

### 第 10 題〔多選．情境題〕
**考點：演算法偏見與公平性（L23402）**

某銀行的貸款審核模型，對兩個「條件相近」的族群產生了大幅不同的核准率，引發公平性疑慮。下列哪些是 **負責任 AI（Responsible AI）** 的正確處理方向？（**多選**）

- (A) 檢查訓練資料是否存在 Data Bias 或 Historical Bias
- (B) 檢視所使用的 Feature 是否隱含對特定族群的歧視
- (C) 只要整體 Accuracy 夠高就不必理會群體差異
- (D) 使用公平性指標分別評估不同群體的模型表現
- (E) 必要時重新蒐集資料或調整、重新訓練模型

---

## 作答格式

請複製以下欄位並填入答案與簡短理由：

```text
第 1 題：（答案）　理由：
第 2 題：（答案）　理由：
第 3 題：（答案）　理由：
第 4 題：（答案）　理由：
第 5 題：（答案）　理由：
第 6 題：（答案）　理由：
第 7 題：（答案，可多選）　理由：
第 8 題：（答案）　理由：
第 9 題：（答案）　理由：
第 10 題：（答案，可多選）　理由：
```

---

## 批改說明

作答完成後，請將本 Issue 的 **label 改為 `ai-analysis`**，系統／講師將據此進行批改，並產出逐題錯題分析與複習建議。

**Comment by JohnnyLai92** [comment]
改用 checklist 勾選題型（#83 / PR #84），舊的文字填答格式練習題直接結案，下次排程會以新格式出題。
